Dockerfile

```diff
@@ -31,7 +33,9 @@ WORKDIR /native
 # Copying jni headers to java /include
 RUN cp -r jniheaders/include $JAVA_HOME
 
+RUN mkdir /out
+
 # Expose
 WORKDIR /native
-ENTRYPOINT ["./gradlew"]
-CMD ["buildProject", "--no-daemon"]
+
+CMD ./gradlew --no-daemon clean buildProject ; cp -R ./build/libs/native-* /out/
```

Comment:
for some odd reason, checksums of build artefacts on different OSs are not matching if the `build` folder in a docker container is mounted to the host's folder. That's why this trick with an extra folder ( `/out`, which is actually the one that's mounted ) is used here